Python/_warnings.c

```diff
@@ -1031,28 +1031,31 @@ get_source_line(PyInterpreterState *interp, PyObject *module_globals, int lineno
     return source_line;
 }
 
+/*[clinic input]
+warn_explicit as warnings_warn_explicit
+
+    message: object
+    category: object
+    filename: unicode
+    lineno: int
+    mod: object = NULL
```

Comment:
```suggestion
    module as mod: object = NULL
```
This should work or am I missing something ?

Reply:
> Error in file "Python/_warnings.c" on line 1053:
> You can't have two parameters named 'module'!

Reply:
That sounds like a clinic bug, it should compare c level names as `module` is implicitly positional only as the first arg.

Reply:
I've sorted out that `module` is converted into `self` on early stages, so I've commited whatever is necessary.

The only caveat is two same named `module` parameters in a docstring, but I believe it's quite of no importance.

Reply:
As I said in https://github.com/python/cpython/pull/92891#discussion_r875466153, I still believe we should fix the AC bug first, and then proceed with this PR. Instead of adding hacks to work around limitations in AC, we should fix AC.

Reply:
- gh-94246

Reply:
@arhadthedev: the issue you opened does not address the problems with this PR.